tests/test_communication_manager.py

```diff
@@ -67,15 +73,14 @@ def test_communication_manager_fails_if_not_configured(new_session_event):
     communication_manager = CommunicationManager()
     try:
         communication_manager.inform(
-            MessageEventTypes.NEW_SESSION.name,
+            RawEventTypes.NEW_MESSAGE.name,
             new_session_event
         )
         raise Exception("This should have raised an exception")
     except Exception as e:
-        assert str(e) == "Dataset Manager has not been set. Communication \
-             Manager needs to be configured first!"
+        assert str(e) == "Dataset Manager has not been set. Communication Manager needs to be configured first!"
```

Comment:
y tho

Reply:
Before it wouldn't pass the pytest since the whitespace gets read differently. Now it passes the pytest. Idk man.

Reply:
Error?

Reply:
Just said they weren't equal to each other, and the strings were identical except for whitespace.